templates/BenchmarkDotNet.Templates.csproj

```diff
@@ -24,4 +24,18 @@
     <Compile Remove="**\*" />
     <None Remove="*.bat;.sh" />
   </ItemGroup>
+
+  <Target Name="Fix BenchmarkDotNetVersion"
+          BeforeTargets="Build">
+    <ReplaceFileText
+            InputFilename="templates/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/.template.config/template.json"
+            OutputFilename="templates/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/.template.config/template.json"
+            MatchExpression='("defaultValue": )"([\d+\.?]+)"'
+            ReplacementText='${1}"$(Major).$(Minor).$(Revision)"' />
```

Comment:
With the recent changes, we should use `$(VersionPrefix)` instead of `$(Major).$(Minor).$(Revision)`.

Reply:
fixed at c6bd682510b877463587bbf752b0e78acb07f035